fastapi_backend/tests/test_email.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+from pathlib import Path
+from fastapi_mail import ConnectionConfig, MessageSchema
+from app.email import get_email_config, send_reset_password_email
+from app.models import User
+
+
+@pytest.fixture
+def mock_settings(mocker):
+    mock = mocker.patch("app.email.settings")
+    # Set up mock settings with test values
+    mock.MAIL_USERNAME = "test_user"
+    mock.MAIL_PASSWORD = "test_pass"
+    mock.MAIL_FROM = "test@example.com"
+    mock.MAIL_PORT = 587
+    mock.MAIL_SERVER = "smtp.test.com"
+    mock.MAIL_FROM_NAME = "Test Sender"
+    mock.MAIL_STARTTLS = True
+    mock.MAIL_SSL_TLS = False
+    mock.USE_CREDENTIALS = True
+    mock.VALIDATE_CERTS = True
+    mock.TEMPLATE_DIR = "email_templates"
+    mock.FRONTEND_URL = "http://test-frontend.com"
+    return mock
+
+
+@pytest.fixture
+def mock_user():
+    return User(
+        email="user@example.com",
+    )
+
+
+def test_get_email_config(mock_settings):
+    config = get_email_config()
+
+    assert isinstance(config, ConnectionConfig)
+    assert config.MAIL_USERNAME == "test_user"
+    assert config.MAIL_PASSWORD == "test_pass"
+    assert config.MAIL_FROM == "test@example.com"
+    assert config.MAIL_PORT == 587
+    assert config.MAIL_SERVER == "smtp.test.com"
+    assert config.MAIL_FROM_NAME == "Test Sender"
+    assert config.MAIL_STARTTLS is True
+    assert config.MAIL_SSL_TLS is False
+    assert config.USE_CREDENTIALS is True
+    assert config.VALIDATE_CERTS is True
```

Comment:
use assert and assert not instead of is True and is False.
`is` should be used only for object identity checks.

Reply:
makes sense.